docs/slow-network-test.rst

```diff
@@ -0,0 +1,62 @@
+Testing Raiden on a slow network
+================================
+
+In order to analyze Raiden performance on slow networks, it is useful to have a way of
+introducing latencies. On Linux this can be achieved by making use of the tc utility.
+
+The following script can be used to prepare the interfaces:
+
+.. code:: bash
+
+    #!/usr/bin/env bash
+
+    IFACE="wlan0"
```

Comment:
Maybe add a comment that this needs to be changed to whichever interface is the default one in the user's machine.

Reply:
Done.